server/handlers/k8sconfig_handler.go

```diff
@@ -16,11 +17,11 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 
 	"github.com/gofrs/uuid"
-
 	"github.com/layer5io/meshery/server/helpers"
 	"github.com/layer5io/meshery/server/models"
 	"github.com/layer5io/meshery/server/models/pattern/core"
 	"github.com/layer5io/meshkit/models/meshmodel"
+	meshmodelcore "github.com/layer5io/meshkit/models/meshmodel/core/v1alpha1"
```

Comment:
oh, my goodness. Now, there is a package name with "core" in it.

Reply:
I should have put a smiley face on this comment to show my sarcasm. No sweat. This package name is fine (until larger changes come in the future).